engine/api/group.go

```diff
@@ -129,6 +130,10 @@ func updateGroupHandler(w http.ResponseWriter, r *http.Request, db *gorp.DbMap,
 	}
 	defer tx.Rollback()
 
+	if strings.HasSuffix(updatedGroup.Name, sdk.DefaultGroupOnProjectSuffix) {
+		return sdk.WrapError(sdk.ErrWrongRequest, "You can create a group with ending with %s", sdk.DefaultGroupOnProjectSuffix)
```

Comment:
i don't understand, if the group ends with the PRJ suffix, we return an error telling him he can do it ?